2.0/README.md

```diff
@@ -0,0 +1,500 @@
+ The command creates a line segment that begins at the point `(cX,cY)` and ends at the point `(pX,pY)` where:
+
+```
+pX = cX + dX
+```
+
+and
+
+```
+pY = cY + dY
+```
+
+After this command the cursor is at `(pX,pY)`.
+
+The command count for a `LineTo` command MUST be the number of pairs of `ParameterIntegers` that follow it, and MUST be greater than zero.
+
+#### 4.3.3.3. ClosePath Command
+
+The `ClosePath` command requires no parameters. The command creates a line segment that begins at the point `(cX,cY)` and ends at the point `(pX,pY)`, where `pX` and `pY` are the absolute coordinates resulting from the most recent MoveTo command.
+
+This command does not change the cursor position.
+
+A `ClosePath` command MUST have a command count of 1.
+
+#### 4.3.4. Geometry Types
+
+The `geometry` field is described in each feature by the `type` field which must be a value in the enum `GeomType`. The following geometry types are supported:
+
+* UNKNOWN
+* POINT
+* LINESTRING
+* POLYGON
+
+Geometry collections are not supported.
+
+##### 4.3.4.1. Unknown Geometry Type
+
+The specification purposefully leaves an unknown geometry type as an option. This geometry type encodes experimental geometry types that an encoder MAY choose to implement. Decoders MAY ignore any features of this geometry type.
+
+##### 4.3.4.2. Point Geometry Type
+
+The `POINT` geometry type encodes a point or multipoint geometry. The geometry command sequence for a point geometry MUST consist of one or more `MoveTo` commands:
+
+```
+PointGeometry = MoveTo+
+```
+
+If the command sequence for a `POINT` geometry type includes only a single `MoveTo` command, then the geometry MUST be interpreted as a single point; otherwise the geometry MUST be interpreted as a multipoint geometry, wherein each `MoveTo` signals a single point.
+
+##### 4.3.4.3. Linestring Geometry Type
+
+The `LINESTRING` geometry type encodes a linestring or multilinestring geometry. The geometry command sequence for a linestring geometry MUST consist of one or more repetitions of a `MoveTo` command followed by one or more `LineTo` commands:
+
+```
+LinestringGeometry = (MoveTo LineTo+)+
+```
+
+If the command sequence for a `LINESTRING` geometry type includes only a single `MoveTo` command then the geometry MUST be interpreted as a single linestring; otherwise the geometry MUST be interpreted as a multilinestring geometry, wherein each `MoveTo` signals the beginning of a new linestring.
+
+##### 4.3.4.4. Polygon Geometry Type
+
+The `POLYGON` geometry type encodes a polygon or multipolygon geometry, each polygon consisting of an exterior ring containing zero or more interior rings. The geometry command sequence for a polygon consists of one or more repetitions of an `ExteriorRing` followed by zero or more `InteriorRing`s. Both `ExteriorRing`s and `InteriorRing`s MUST consist of one or more repetitions of a `MoveTo` command, followed by two or more `LineTo` commands, followed by exactly one `ClosePath` command:
```

Comment:
> polygon consists of one or more repetitions of an `ExteriorRing` followed by zero or more `InteriorRing`s

It's a single polygon, it should consist of _exactly one_ `ExteriorRing` followed by zero or more `InteriorRing`s, right?